docs/content/doc/usage/backup-and-restore.en-us.md

```diff
@@ -95,3 +95,37 @@ Repository Git Hooks should be regenerated if installation method is changed (eg
 With Gitea running, and from the directory Gitea's binary is located, execute: `./gitea admin regenerate hooks`
 
 This ensures that application and configuration file paths in repository Git Hooks are consistent and applicable to the current installation. If these paths are not updated, repository `push` actions will fail.
+
+### Using Docker (`restore`)
+
+There is also no support for a recovery command in a Docker-based gitea instance. The restore process contains the same steps as described in the previous section but with different paths.
+
+Example:
+
+```sh
+# open bash session in container
+docker exec --user git -it 2a83b293548e bash
+# unzip your backup file within the container
+unzip gitea-dump-1610949662.zip
+cd gitea-dump-1610949662
+# restore the gitea data
+mv data/* /data/gitea
+# restore the repositories itself
+mv repos/* /data/git/repositories/
+# adjust file permissions
+chown -R git:git /data
+# Regenerate Git Hooks
+/app/gitea/gitea -c '/data/gitea/conf/app.ini' admin regenerate hooks
```

Comment:
Actually, the default Docker image (with root) and Docker rootless image is different.

See: https://github.com/go-gitea/gitea/blob/main/Dockerfile.rootless

```
ENV GITEA_WORK_DIR /var/lib/gitea
ENV GITEA_APP_INI /etc/gitea/app.ini
```

Reply:
Maybe it's worth splitting the docker section again into `root` and `rootless section` in a following PR?

Reply:
I'm also wondering now it the restore should respect the changes done in #17846 ?

`/app/gitea/gitea` 👉 `/usr/local/bin/gitea`

Reply:
When using `/usr/local/bin/gitea`, then users do not need to provide config file in command line.

However ..... the `data` paths are still different between Docker-root and Docker-rootless 😂

Reply:
Oh and the `GITEA_CUSTOM` is nested differently:

https://github.com/go-gitea/gitea/blob/f1b1472632e36c3bb5df73f184a6efeef47f5d01/Dockerfile#L56-L57

https://github.com/go-gitea/gitea/blob/f1b1472632e36c3bb5df73f184a6efeef47f5d01/Dockerfile.rootless#L58-L63

Maybe the backup tar looks different in rootless container

Reply:
stared working on docker rootless restore docs and ran into #18877

Reply:
Just added another docs section for docker-rootless restore